modules/ding_groups/ding_groups.pages_default.inc

```diff
@@ -173,11 +121,33 @@ function ding_groups_default_page_manager_handlers() {
       'css_class' => 'ding-group-events',
     );
     $pane->extras = array();
-    $pane->position = 2;
+    $pane->position = 1;
     $pane->locks = array();
     $pane->uuid = '3bc42ce7-ca5f-4e5c-9176-2bcbaecc961a';
     $display->content['new-3bc42ce7-ca5f-4e5c-9176-2bcbaecc961a'] = $pane;
-    $display->panels['main_content'][2] = 'new-3bc42ce7-ca5f-4e5c-9176-2bcbaecc961a';
+    $display->panels['attachment_1_1'][1] = 'new-3bc42ce7-ca5f-4e5c-9176-2bcbaecc961a';
+    $pane = new stdClass();
+    $pane->pid = 'new-fff67ee4-207e-421d-9fd6-ad12b8078621';
+    $pane->panel = 'left_sidebar';
+    $pane->type = 'block';
+    $pane->subtype = 'og_menu-og_single_menu_block';
```

Comment:
Sorry, that was a slip up on my side, i thought "og" was the danish for "and" but i see now that it's not :)